modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/ExecutionTest.java

```diff
@@ -980,6 +980,71 @@ public void testCorrelatedNestedLoopJoin() {
         }
     }
 
+    /** */
+    @Test
+    public void testMergeJoin() {
+        ExecutionContext<Object[]> ctx = executionContext(F.first(nodes()), UUID.randomUUID(), 0);
+        IgniteTypeFactory tf = ctx.getTypeFactory();
+        RelDataType rowType = TypeUtils.createRowType(tf, int.class, String.class, int.class);
+
+        int[] leftSizes = {1, 99, 100, 101, 512, 513, 2000};
```

Comment:
plz change hardcoded, like it made here
https://github.com/apache/ignite/blob/ignite-12248/modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/LimitOffsetTest.java#L212

Reply:
fixed